src/views/Table.vue

```diff
@@ -18,6 +18,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 <template>
   <div class="h-100">
     <TableComponent
+      :tasksFilter="tasksFilter"
```

Comment:
Rather than do this, I think it would make sense to extract the filtering out of the component (`src/components/cylc/table/Table.vue`) into the view (`src/views/Table.vue`)

Reply:
Sounds reasonable, we should follow the same approach with all views.